model/config.go

```diff
@@ -519,11 +519,6 @@ func (o *Config) SetDefaults() {
 		o.FileSettings.AmazonS3Endpoint = "s3.amazonaws.com"
 	}
 
-	if o.FileSettings.AmazonS3Region == "" {
-		// Defaults to "us-east-1" region.
-		o.FileSettings.AmazonS3Region = "us-east-1"
```

Comment:
Do we still want this?  What happens when the region is blank?  does it connect to `us-east-1` by default?